src/main/resources/db/migration/V005__Process_events.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE process_events (
```

Comment:
We are creating process_events table and then copying all values from envelope_state into process_events.Any reason why we are not just renaming envelope_state to process_events in migratiion script ?

Reply:
not backwards compatible. firstly additions, then deletion